magic-modules/merge-pr.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+set -x
+
+echo "$CREDS" > ~/github_private_key
```

Comment:
How about using a volume to keep the private key in a single place?

Reply:
There are some recommended options here: https://concourse.ci/creds.html

This solution is a stepping stone to using Vault or CredHub, and is the recommended method by Concourse (I believe).

Reply:
OK.